src/index.test.ts

```diff
@@ -11,5 +13,13 @@ describe('environment', () => {
     Object.values(environment.parsers).forEach(parser => {
       expect(typeof parser).toBe('function');
     });
+
+    Object.entries(parsers).forEach(([name, parser]) => {
+      if (typeof parser === 'function') {
+        expect(environment.parsers).toEqual(
+          expect.objectContaining({ [name]: parser }),
+        );
+      }
+    });
```

Comment:
This test ensures every parser exported by `parsers.ts` has been exposes in `environment.parsers` object.
The test above can only ensure the items on `environment.parsers` are functions.

If a new parser is added to `parsers.ts` and exported, but `environment.parsers` forgets to expose it, this test would fail, but the test above would not.

Reply:
I understand now, thanks.